add hashflow bnb to dex trades - ready to review

henrystats opened the pull request:
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

dune-eng commented:
Workflow run id 3893084222 approved.

dune-eng commented:
Workflow run id 3894268976 approved.

dune-eng commented:
Workflow run id 3894268997 approved.

Hosuke reviewed (approved):
This PR adds `hashflow_bnb_trades`. I have reviewed the PR and here is the summary:

- Metadata & naming conventions look good ✅
- Config blocks match uniqueness tests in schema files ✅
- Tests and seeds added for the new models ✅
- Incremental filters added when joining large tables ✅
- Contract addresses are lowercased ✅
- Inner join to large tables(bnb.transactions) as possible ✅

[Sample query results](https://dune.com/queries/1888939?d=1)
Great work @henrystats

dune-eng commented:
Workflow run id 3919034889 approved.

dune-eng commented:
Workflow run id 3919034888 approved.

henrystats commented:
@Hosuke Antonio mentioned that some new ABIs were submitted for hashflow (when I was working on this pr I realised that the tables stopped updating after October). 

There's a new event (xchain  trade) and the other event tables are also now updating. 

I added for both avalanche_c & bnb models and added new seeds for both as well.

Hosuke reviewed (approved):
Thank you @henrystats,
LGTM. for Pool_evt_XChainTrade updates

jeff-dude reviewed (approved):
thank you @henrystats 🤝
